### DISCUSSION TITLE
Exported CSV data: Checkmarks.csv, meanings of the values in the 2nd Column

### DISCUSSION THREAD

**Discussion by gt7599a** [iSoron/uhabits#1157] [Help & FAQ] [answered]
Seeking to clarify for myself & hopefully help other users.
In a CSV export, within the Checkmarks.csv the first column has date, the second column has numbers from 0-2.
It _appears_ that:
- 2 means "I did whatever that habit today."
- 1 means "I'm still fulfilling my Habit goal even though I haven't done this today." 
- 0 (I'm guessing) means "I marked it as "done" & then later un-marked it."

Is this correct?  
If not, could someone correct me & clarify what these values stand for?

Thanks so much.

**Comment by kenxle** [comment]
I also see values of 2, -1, and 0. From what I can tell, 2 is the only one that represents "checked." Any help on the difference between -1 and 0?

**Comment by iSoron** [comment, marked as answer]
Here is a complete description of each possible value:
https://github.com/iSoron/uhabits/blob/64337b9bee2fa03f9e7144c1aaa9fe5103858247/uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/models/Entry.kt#L26-L51

**Comment by gt7599a** [comment]
Thank you both, this is very helpful.

Ed
